case-studies/bayren.html

```diff
@@ -0,0 +1,20 @@
+---
+layout: case-study
+
+client: bayren
+
+title: Bay Area Regional Energy Network (BayREN)
+
+tags: [Our Work]
+
+description: "An example of our work – an eTool - for the Bay Area Regional Energy Network."
+
+client-text: The BayREN Codes and Standards program helps Bay Area counties reduce their energy usage through improved building design and construction. The collaborative program supplies local governments with the resources they need to achieve full compliance with and enforce the California Energy Code, including best practice guides, checklists, policy support, targeted training, and on-site support.
+
+challenge-text: As a component of a Multifamily Subprogram and the Codes and Standards Subprogram to provide services to the members of BayREN, the Association of Bay Area Governments (ABAG) was interested in options regarding how to implement an online or at permit counter mobile system through which permit applicants answer a set of questions to determine their energy requirements for the following permit types:<ol class="font-size-md font-family-lato line-height-2"><li>Residential Water Heating Replacement</li><li>Residential Roof Replacement</li><li>Residential Window Replacement</li></ol>
+
+
+technologies: [javascript, angular, node, docker, kubernetes, mysql]
+
+outcome-text: "We built a set of tools which takes AutoCAD files, identifies the layers that show certain required information like booth dimensions that, when uploaded, are saved as sets of vertices and redrawn dynamically. We built an online app to show what the floor plan and booth spaces would look like ahead of each event. We were able to create these custom floor plans by manipulating CAD files and sets of vertices which was also rendered on the front-end. This floorplan application allowed buyers to know the “address” of a booth they wanted to visit. The dynamic display would instantly update who moved, who is at this spot or that booth. It made it easy for buyers at market to get an overall look at the layout of the floor. Here’s an example: If someone cancelled from appearing at market their booth would be removed right away so buyers intending to visit it wouldn’t be disappointed."
```

Comment:
This line looks like it's from the case study on WEM.

Reply:
That's a good point. That's what it said in the spec but probably a mistake on John's part. I'll ask him for the correct text.

Reply:
Updated with the correct text.